app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -2,6 +2,7 @@ package org.fossasia.openevent.general.attendees
 
 import android.arch.lifecycle.MutableLiveData
 import android.arch.lifecycle.ViewModel
+import android.view.View
```

Comment:
This should not be in the ViewModel. Send a boolean from the viewModel and update the visibility in the fragment.
Just like it is being done in the `showProgress` method